tests/examples/pacer/case_queries/case_query_error.json

```diff
@@ -8,5 +8,10 @@
   "date_last_filing": "2020-03-06",
   "date_terminated": "2020-03-06",
   "docket_number": null,
+  "federal_defendant_number": null,
+  "federal_dn_case_type": null,
+  "federal_dn_judge_initials_assigned": null,
+  "federal_dn_judge_initials_referred": null,
+  "federal_dn_office_code": null,
   "related_bankruptcy": "19-30334"
```

Comment:
this one uses `20-03006` - and it returns no result.  is there a reason this doesnt bring back office code?

Reply:
Well actually the example was wrong, I fixed it so the docket_number can be parsed properly, thanks for noticing it.